src/lib/bootloader/device_map.rb

```diff
@@ -118,14 +119,24 @@ def order_boot_device
       # want to modify its MBR. So we get disk of such partition and change order to add it
       # to top of device map. For details see bnc#887808,bnc#880439
       boot_disk = Yast::BootStorage.disk_with_boot_partition
-      priority_disks = ::Bootloader::Stage1Device.new(boot_disk).real_devices
+      priority_disks = ::Bootloader::Stage1Device.new(boot_disk.name).real_devices
       # if none of priority disk is hd0, then choose one and assign it
       return if any_first_device?(priority_disks)
 
       change_order(priority_disks.first)
     end
 
     def fill_mapping
+      # storage-ng
+      # BIOS-ID is not supported in libstorage-ng, so let's simply create a
+      # mapping entry per disk for the time being (see commented code for the
+      # real expected behavior)
+      staging = Y2Storage::StorageManager.instance.staging
+      staging.disks.each_with_index do |disk, index|
+        add_mapping("hd#{index}", disk.name)
+      end
+# rubocop:disable Style/BlockComments
```

Comment:
please remove whole this commented out code, as we do not plan to implement it, so lets use just common order.